examples/widgets_example.hpp

```diff
@@ -68,8 +68,6 @@ namespace WidgetsExample
             imageTextButton->setAlignment(gcn::ImageTextButton::TOP);
             imageTextButton->addActionListener(this);
 
-            imageButton->setSize(220, 90);
-            imageButton->addActionListener(this);
```

Comment:
Same as line 63-64, so the removal. (and having actionlistener twice made click callback called twice :/ )